src/browserify/ReactForBrowserify.js

```diff
@@ -6,23 +6,21 @@
  * LICENSE file in the root directory of this source tree. An additional grant
  * of patent rights can be found in the PATENTS file in the same directory.
  *
- * @providesModule React
+ * @providesModule ReactForBrowserify
  */
 
 'use strict';
 
 var ReactDOM = require('ReactDOM');
 var ReactDOMServer = require('ReactDOMServer');
-var ReactIsomorphic = require('ReactIsomorphic');
+var React = require('React');
 
 var assign = require('Object.assign');
 
 // `version` will be added here by ReactIsomorphic.
```

Comment:
This comment is outdated now, is it not? Should be `// `version` will be added here by React module.` or something.